### PR TITLE
english

### PR DESCRIPTION
**Pull request by hYdos**

### CODE REVIEW

File: src/main/java/com/lunarclient/bukkitapi/object/StaffModule.java [hunk 1]
```diff
@@ -6,7 +6,7 @@
  *
  * Currently only XRAY is implemented but there are
  * future plans to expand this to possibly NAME_TAGS,
- * and there is still discussion of weather BUNNY_HOP
+ * and there is still discussion of wheather BUNNY_HOP
```
Comment (NickAcPT) [inline src/main/java/com/lunarclient/bukkitapi/object/StaffModule.java:9]: whether

**Review by NickAcPT — APPROVED** [review]
LGTM

**Review by burgerguy — APPROVED** [review]
Looks good to me!

**Comment by PrestigePvP** [comment]
Woopsie :)

**Pull request closed without merging**